services/movie.py

```diff
@@ -0,0 +1,36 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
```

Comment:
It accepts Optional[list[int]] because have as default None
Fix in other places